src/app.js

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios';
+
+//form aanspreken en eventlistener op geplaatst. Als er wordt gesubmit/geentered, dan functie fetchcountries uitvoeren.
+const formElement = document.getElementById('receivingInput');
+formElement.addEventListener('submit', fetchCountries);
+
+//VRAAG: waarom verdwijnt de data zo snel? :(
+
+//variabele aangemaakt voor het laatste gedeelte van de uri.
+const INPUT = document.getElementById('inputfield')
+
+//in deze functie wordt de endpoint gebruikt en de data van het ingevoerde land gebruikt. Van die data wordt de name, flag, subregion,
+// population, capital en currencies via innerhtml op de pagina geplaatst. bij die laatste wordt er eerst een if functie uitgevoerd om te
+// kijken of er één of twee currencies zijn.
+
+async function fetchCountries() {
+    try {
+        const result = await axios.get('https://restcountries.com/v2/name/' + INPUT.value);
+        const country = result.data[0];
```

Comment:
Hier zou je nog spreading kunnen toepassen, zodat je niet overal `country.` voor hoeft te zetten
```suggestion
        const { name, flag, population, subregion, capital } = result.data[0];
```